user/howto/add-elk-stack-to-an-application.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: user-doc
+title: Add ELK Stack to an Application
+---
+
+This page details how to add [Elasticsearch](https://www.elastic.co/products/elasticsearch),
+[Logstash](https://www.elastic.co/products/logstash) and [Kibana](https://www.elastic.co/products/kibana)
+- the Elastic stack or ELK stack - to an application.
+
+# Elasticsearch Setup
+
+1. Add a new platform for Elasticsearch using the 'Elasticsearch with LB' pack.
+
+1. If required, edit the configuration under the Elasticsearch component.
```

Comment:
Like Change values for Number of shards or Number of replicas or change default directory location for installation,Elasticsearch logs, etc..